src/components/common/user-avatar/user-avatar.tsx

```diff
@@ -32,22 +64,32 @@ const UserAvatar: React.FC<UserAvatarProps> = ({ name, photo, size, additionalCl
     }
   }, [size])
 
+  const { error, value, loading } = useAsync(async (): Promise<NameAndPhoto> => {
+    if (username) {
+      user = await getUser(username)
+    }
+    return {
+      name: user?.displayName ?? t('common.guestUser'),
+      photo: user?.photo ?? `${customizeAssetsUrl}img/avatar.png`
+    }
+  }, [user, username, t])
+
   return (
-    <span className={'d-inline-flex align-items-center ' + additionalClasses}>
-      <Image
-        src={photo}
-        className={`rounded`}
-        alt={t('common.avatarOf', { name })}
-        title={name}
-        height={imageSize}
-        width={imageSize}
-        layout={'fixed'}
-      />
-      <ShowIf condition={showName}>
-        <span className={`ml-2 mr-1 ${styles['user-line-name']}`}>{name}</span>
-      </ShowIf>
-    </span>
+    <ShowIf condition={!error && !loading}>
```

Comment:
How about using AsyncLoadingBoundary?

Reply:
Isn't  that a new thing you just intorduced?

Reply:
Yes, this didn't exist in the main code until yesterday evening.